src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1541,51 +1541,6 @@ public static IEnumerable<string> ParseConditionalCompilationSymbols(string valu
             return defines.AsEnumerable();
         }
 
-        public static bool TryParseLanguageVersion(string value, out LanguageVersion version)
```

Comment:
Nevermind. It was refactored and it seems like this is a leftover:
https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/LanguageVersion.cs
#Closed